src/System.Security.Cryptography.ProtectedData/src/project.json

```diff
@@ -0,0 +1,28 @@
+{
+  "frameworks": {
+    "netstandard1.4": {
+      "imports": [ "dotnet5.5" ],
+      "dependencies": {
+        "Microsoft.NETCore.Platforms": "1.0.1-rc3-23908",
+        "System.Diagnostics.Debug": "4.0.10",
+        "System.Diagnostics.Tools": "4.0.0",
+        "System.IO": "4.0.10",
+        "System.Resources.ResourceManager": "4.0.0",
+        "System.Runtime": "4.0.20",
+        "System.Runtime.Extensions": "4.0.0",
+        "System.Runtime.InteropServices": "4.0.20",
+        "System.Security.Cryptography.Algorithms": "4.1.0-rc3-23908"
+      }
+    },
+    "net46": {
+      "dependencies": {
+        "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.1"
+      }
+    },
+    "net461": {
```

Comment:
Deleted the 4.6.1 section (as part of the syncup to the latest master branch changes)